.tekton/scripts/check-partner-tasks.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPTDIR/../.." || exit 1
+
+if [ ! -e "partners/" ]; then
+    echo "No partners directory exists. Skip checks against partner tasks."
+    exit 0
+fi
+
+check_dir_structure() {
+    resultf=$(mktemp)
+    check_result=$(mktemp)
+
+    for task_dir in partners/*; do
+        owners_file="$task_dir/OWNERS"
+        if [ ! -e "$owners_file" ]; then
+            echo "error: missing file $owners_file" >>"$check_result"
```

Comment:
Do you think separating the `OWNERS` file check with the directory structure will be good idea? Also if the checking if the OWNERS file is empty or not?

Reply:
No separation.

> Also if the checking if the OWNERS file is empty or not?

I have no idea. But, in order to keep the consistency, this file should exist at least.

Reply:
I think the partner engineer should atleast provide their github user id inside the OWNERS file so that in case any future communication required, we can use that id. Empty OWNERS file only for the sake of passing the validation does not make sense to me. WDYT?

Reply:
I don't want to make this assumption.

Reply:
I was trying to think of what could be the use case for OWNERS file in case of partner task submission. Should contain some information that will be useful for us?